### ISSUE TITLE
Answers ?

### ISSUE THREAD

**Issue by hexadeciman** [gdi2290/ngExam#2]
It would be great to have a file with multiple possible answers to each question.

**Comment by PatrickJS** [comment]
another great idea thanks

**Comment by hexadeciman** [comment]
Can I make a pull request with some of my answers ? (I don't think they are all correct && complete)

**Comment by PatrickJS** [comment]
yeah create an answers folder and a folder of the question (as a number for now) perhaps with your username. What do you suggest should be the format?

answers
⦜1
  ⦜ overben.js

**Comment by PatrickJS** [comment]
you can keep it anonymous if you want until I figure out a system for people to submit their answers either by pull requests or online forms

**Comment by tylerdmace** [comment]
I think having a directory called `answers` and markdown files for each question should suffice. Then link to the file from the question in the `readme.md`.

**Example:** answer for question 1 would be contained in `answers/1.md`

**Comment by dweitz43** [comment]
@gdi2290  just wondering if any progress was ever made on collecting answers for any of these questions? I know there is a gist linked for one of the questions...if not, i can definitely try to create a pr with some of my answers if they are acceptable.

**Comment by PatrickJS** [comment]
can you create a pr and create another file with the answers

**Comment by dweitz43** [comment]
Sure, I'll start working on that today

**Comment by dweitz43** [comment]
@gdi2290 #9 :)
